src/main/webapp/app/exercises/shared/participation/participation.utils.ts

```diff
@@ -0,0 +1,12 @@
+import { ProgrammingExerciseStudentParticipation } from 'app/entities/participation/programming-exercise-student-participation.model';
+import { createBuildPlanUrl } from 'app/exercises/programming/shared/utils/programming-exercise.utils';
+import { ProfileInfo } from 'app/shared/layouts/profiles/profile-info.model';
+import { ProgrammingExercise } from 'app/entities/programming-exercise.model';
```

Comment:
Here is still a Codacy issue about unsused code.

Reply:
I hate my life...